TROUBLESHOOTING.md

```diff
@@ -0,0 +1,52 @@
+# Troubleshooting
+
+## "ALPN is not configured properly"
+
+This exception usually indicates that Netty isn't set up properly.
+
+If using gRPC version 1.19.0 or newer, consider using `io.grpc:grpc-netty-shaded`
+dependency so that it does not conflict with other packages that also require Netty.
+
+If your gRPC dependency is older than 1.19.0, consider updating.
+As of 0.35.0, packages in this `google-cloud-java` already declares dependency on `grpc-netty-shaded`.
```

Comment:
s/in this/in/, s/already declares dependency/already declare a dependency/

Reply:
Done